src/Toast.php

```diff
@@ -0,0 +1,402 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+use JsonException;
+use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Html\Html;
+
+use function array_merge;
+
+/**
+ * Toasts renders an toast bootstrap component.
+ *
+ * For example,
+ *
+ * ```php
+  * Toast::widget()
+ *     ->title('Hello world!')
+ *     ->dateTime('a minute ago')
+ *     ->body('Say hello...')
+ * ])
+ * ->begin();
+ * ```
+ *
+ * The following example will show the content enclosed between the [[begin()]]
+ * and [[end()]] calls within the toast box:
+ *
+ * ```php
+ * Toast::widget()
+ *     ->title('Hello world!')
+ *     ->dateTime('a minute ago')
+ * ])
+ * ->begin();
+ *
+ * echo 'Say hello...';
+ *
+ * Toast::end();
+ * ```
+ *
+ * @see https://getbootstrap.com/docs/components/toasts/
+ *
+ * @author Simon Karlen <simi.albi@outlook.com>
+ */
+final class Toast extends Widget
+{
+    /**
+     * @var string the body content in the alert component. Note that anything between
+     * the [[begin()]] and [[end()]] calls of the Toast widget will also be treated
```

Comment:
* the {@see begin()} and {@see end()} calls of the Toast widget will also be treated